pkg/cache/node.go

```diff
@@ -149,10 +149,13 @@ func (n *SchedulerNode) handleNodeRecovery(event *fsm.Event) {
 		zap.String("nodeID", n.name),
 		zap.Bool("schedulable", n.schedulable))
 
-	request := &si.UpdateRequest{
+	allocRequest := &si.AllocationRequest{
 		Asks:     nil,
 		Releases: nil,
-		NewSchedulableNodes: []*si.NewNodeInfo{
+		RmID:     conf.GetSchedulerConf().ClusterID,
+	}
```

Comment:
Sending this would be a nop in the core. The core checks for the length of Asks and Releases and if they are zero (0) we just return without doing anything. See `handleRMUpdateAllocationEvent()`

Reply:
Just retained those request params as it was there before. Removed the same now.